benchmark/sirun/goal.json

```diff
@@ -3,7 +3,7 @@
   "async_hooks": {
     "no-hooks": {
       "instructions": 6562157820,
-      "nodeVersion": "16.12.0",
+      "nodeVersion": "18.18.0",
```

Comment:
@bengl When updating the nodeVersion here, do we also need to update the summary values?

Reply:
Probably? It's been a long time since I worked on this. @tlhunter may be able to better answer.

Reply:
I don't think this file is used by the benchmark runner. Pretty sure it's something that predates the CI action. Maybe it was used for manual runs?